tracer/src/Datadog.Trace/Debugger/LiveDebugger.cs

```diff
@@ -274,8 +274,10 @@ private void CheckUnboundProbes()
 
         private void AcceptConfiguration(ProductConfigChangedEventArgs args)
         {
+            Log.Information("AcceptConfiguration: Starting ...");
```

Comment:
Do we want to keep this logging here? Seems a bit chatty (these would pop up in logs every 5 seconds I assume?)

Reply:
Or otherwise maybe just reduce log verbosity?

Reply:
Thanks, these were for my own debugging, I deleted most of them, seems these two slipped thought. Fixed in ae8a05e44.

Reply:
Cool, thanks! Note that https://github.com/DataDog/dd-trace-dotnet/commit/ae8a05e44365db8eb096a17cf8d0bb56ea8307f7 didn't include the deletion, maybe you haven't pushed the right commit in yet?